pkg/cmd/codespace/create.go

```diff
@@ -152,6 +142,37 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		return fmt.Errorf("error getting repository: %w", err)
 	}
 
+	a.StartProgressIndicatorWithLabel("Validating repository for codespaces")
```

Comment:
I like there is a descriptive progress indicator here, otherwise the delay between the prompts could have been jarring 👍